indico/web/client/js/utils/react_integration.js

```diff
@@ -0,0 +1,12 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+export function setNativeInputValue(input, value) {
```

Comment:
Iirc, this module isn't in master yet. It's getting a bit hard to track where I've made what change and plan for merges. Anyhow, in the time picker PR, I've added this method to the custom element base class, and that's where I'll be sourcing it. Now that I'm thinking about it, I'll just do a separate PR just for that change so it can trickle down to open ones.